Header header
osi3_bridge/MovingObject[] objects
